scripts/seed.py

```diff
@@ -0,0 +1,192 @@
+# Command to run : python manage.py shell --settings=settings.dev  < scripts/seed.py
+from datetime import timedelta
```

Comment:
What if bring os import above datetime and add a new line in between. this way we can group all `import *` type imports in one place and then `from X import Y` types, in other one.